phys2bids/viz.py

```diff
@@ -41,3 +41,21 @@ def ntr2time(x):
     subplot.plot(time, trigger, '-', time, time, '-')
     plt.savefig(outfile + '_trigger_time.png', dpi=dpi)
     plt.close()
+
+def plot_all(phys_in,infile,outfile,dpi = SET_DPI,size = FIGSIZE):
+    ch_num = len(phys_in.ch_name) # get number of channels:
+    fig,ax = plt.subplots(ch_num-1,1,figsize = size,sharex=True)
+    row = 0
+    time = phys_in.timeseries[0] # assume time is first channel
+    fig.suptitle(infile)
+    for index, timeser in enumerate(phys_in.timeseries[1:]):
+        index += 1
```

Comment:
This too might not be necessary: why don't you use `index` instead of dealing with `row` (if you think it's more clear, use the name`row`), and correct indexing where you need to?